doc/src/manual/performance-tips.md

```diff
@@ -1631,3 +1631,31 @@ will not require this degree of programmer annotation to attain performance.
 In the mean time, some user-contributed packages like
 [FastClosures](https://github.com/c42f/FastClosures.jl) automate the
 insertion of `let` statements as in `abmult3`.
+
+## [Multithreading and linear algebra](@id man-multithreading-linear-algebra)
+
+This section applies to multithreaded Julia code which, in each thread, performs linear algebra operations.
+Indeed, these linear algebra operations involve BLAS / LAPACK calls, which are themselves multithreaded.
+In this case, one must ensure that cores aren't oversubscribed due to the two different types of multithreading.
+
+Julia compiles and uses its own copy of OpenBLAS for linear algebra, whose number of threads is controlled by the environment variable `OPENBLAS_NUM_THREADS`.
+It can either be set as a command line option when launching Julia, or modified during the Julia session with `BLAS.set_num_threads(N)` (the submodule `BLAS` is exported by `using LinearAlgebra`).
+Its current value can be accessed with `BLAS.get_num_threads()`.
+
+When the user does not specify anything, Julia tries to choose a reasonable value for the number of OpenBLAS threads.
+This choice depends on the platform and the Julia version, but checking and setting the value manually is typically a better idea, and here's why:
+
+* If `OPENBLAS_NUM_THREADS=1`, OpenBLAS uses the calling Julia thread(s), i.e. it "lives in" the Julia thread that runs the computation.
+* If `OPENBLAS_NUM_THREADS=N>1`, OpenBLAS creates and manages its own pool of threads (`N` in total). There is just one OpenBLAS thread pool shared among all Julia threads.
+
+When you start Julia in multithreaded mode with `JULIA_NUM_THREADS=X`, it is generally recommended to set `OPENBLAS_NUM_THREADS=1`.
+Given the behavior described above, increasing the number of BLAS threads to `N>1` can very easily lead to worse performance, in particular when `N<<X`.
+However this is just a rule of thumb, and the best way to set each number of threads is to experiment on your specific application.
+
+## [Alternative linear algebra backends](@id man-backends-linear-algebra)
+
+As an alternative to OpenBLAS, there exist several other backends that can help with linear algebra performance.
+Prominent examples include [MKL.jl](https://github.com/JuliaLinearAlgebra/MKL.jl), [AppleAccelerate.jl](https://github.com/JuliaMath/AppleAccelerate.jl) and [Octavian.jl](https://github.com/JuliaLinearAlgebra/Octavian.jl).
```

Comment:
I wouldn't mention Octavian.jl here because AFAIK it doesn't hook into LBT and thus isn't a backend (it doesn't slot in for standard BLAS calls but has differently named functions, e.g. `matmul!` vs `mul!`. Other backends would be [BLISBLAS.jl](https://github.com/JuliaLinearAlgebra/BLISBLAS.jl) and [FujitsuBLAS.jl](https://github.com/giordano/FujitsuBLAS.jl). But, if at all, I would only mention the former (since the latter is pretty much hardware specific).

Reply:
Removed it!